libs/ui/select/brain/src/lib/brn-select-content.component.ts

```diff
@@ -201,9 +196,10 @@ export class BrnSelectContentComponent implements AfterViewInit {
 	}
 
 	public updateArrowDisplay(): void {
-		this.canScrollUp.set(this.viewport.nativeElement.scrollTop > 0);
-		const maxScroll = this.viewport.nativeElement.scrollHeight - this.viewport.nativeElement.clientHeight;
-		this.canScrollDown.set(Math.ceil(this.viewport.nativeElement.scrollTop) < maxScroll);
+		const nativeElement = this.viewport().nativeElement;
```

Comment:
Not a blocker, but you could probably even simplify to something like this if you want.
```suggestion
		const { scrollTop, scrollHeight, clientHeight} = this.viewport().nativeElement;
```

Reply:
Done ;)